packages/palette_generator/example/lib/main.dart

```diff
@@ -189,19 +192,19 @@ class PaletteSwatches extends StatelessWidget {
   ///
   /// The [generator] is optional. If it is null, then the display will
   /// just be an empty container.
-  const PaletteSwatches({Key key, this.generator}) : super(key: key);
+  const PaletteSwatches({Key? key, this.generator}) : super(key: key);
 
   /// The [PaletteGenerator] that contains all of the swatches that we're going
   /// to display.
-  final PaletteGenerator generator;
+  final PaletteGenerator? generator;
 
   @override
   Widget build(BuildContext context) {
     final List<Widget> swatches = <Widget>[];
-    if (generator == null || generator.colors.isEmpty) {
+    if (generator == null || generator!.colors.isEmpty) {
```

Comment:
Please make a local for `generator` so that you don't need to use `generator!` everywhere in this method. (The reason it's happening now is that it's an accessor rather than a local variable.)